src/Kernel.php

```diff
@@ -70,6 +75,8 @@ public function boot(InputInterface $input, OutputInterface $output): void
 
         if (!$hasLoadedPackages) {
             $this->packageImporter->clean();
+        } elseif ($this->composer->hasRun()) {
+            $this->restart();
```

Comment:
This really needed if no `bootstrap.php` is present ?

Reply:
I don't une bootstrap.php in this PR

Reply:
Yep indeed, sorry LGTM some confusion with `autoload` ^^